dotcom-rendering/src/web/layouts/CommentLayout.tsx

```diff
@@ -209,6 +210,12 @@ const minHeightWithAvatar = css`
 	min-height: 259px;
 `;
 
+const straightLinesColour = (format: ArticleFormat) => {
+	if (format.theme === ArticleSpecial.SpecialReportAlt)
+		return transparentColour(neutral[60], 0.3);
+	else return undefined;
+};
```

Comment:
This can be a ternary, but I think it would be better to actually use `palette.border.article` explicitely, all the time!

This means the change can be done entirely within `DecideLines`.